### DISCUSSION TITLE
Tutorial bisa menguasai bahasa pemrograman gimana sih!?

### DISCUSSION THREAD

**Discussion by piansah** [deaafrizal/forum#93] [The Lounge]
gw baru belajar html, css sama javascript. tapi yang gw pelajari itu sekedar liat tutorial dari beberapa channel yt. udah coba buat mini porject juga tapi itu juga sebatas tutorial dari yt (copypaste). apa ada saran selain buat mini project  biar gw bisa menguasai javacript?!
gw belajar frontend sesuai roadmap.sh, sejauh ini udah masih di javascript tapi udah coba di reactjs, gw ragu mau lanjut karena gw belum menguasai vanilla javascript.

**Comment by mwafa** [comment]
gw lebih sering menentukan goal yang real, terus menentukan stack.

jadi gw lebih sering bikin mini project, akhirnya nanti ketemu kasus kasus yang bisa nambah kemampuan coding.
